server/service/src/report/report_service.rs

```diff
@@ -455,15 +455,26 @@ fn wasm_sql(
     })
 }
 
+#[derive(Serialize)]
+struct WasmData {
```

Comment:
I am keen to try and standardies this, rather then manually inserting in context:https://github.com/msupply-foundation/open-msupply/compare/4702-add-report-sorting-via-wasm-functions...4702-add-report-sorting-via-wasm-functions(Use-from-serialise-rather-then-inserting-manually)

This would also mean you don't have to change existing templates (otherwise all reports would break, even existing ones ?)

Reply:
This makes perfect sense and looks neat having

Reply:
Additional keys should always be backwards compatible for JSON/Object data